### PR TITLE
add gitlab ci

### PR DESCRIPTION
**Pull request by phozzy**
add gitlab-ci configuration file and scripts for a build and a test stages.

### CODE REVIEW

File: build.sh [hunk 1]
```diff
@@ -0,0 +1,8 @@
+#!/usr/bin/env bash
+NEWCONTAINER=$(buildah --storage-driver vfs from postgres:${PG_VER})
```
Comment (gijskant) [inline build.sh:2]: no `-alpine`?

Comment (phozzy) [reply build.sh:2]: to be consistent. I use this image as a base image to build an image that I use to compile extension. So we can be sure about compatibility.

**Review by gijskant — APPROVED** [review]
Tested the images and it works. Idea for improvement: base the image on the `-alpine` version, which is ~70MB instead of the ~300MB of this image.
Could you squash your changes into a single commit or a few commits prior to merging?

File: build.sh (hunk 1 above)
Comment (phozzy) [reply build.sh:2]: I took a look at Dockerfile for alpine based image, they use to compile postgres at the build stage.... That is a bit weird for me ¯\_(ツ)_/¯

**Pull request merged**